docs/src/CommutativeAlgebra/GroebnerBases/orderings.md

```diff
@@ -7,10 +7,10 @@ DocTestSetup = Oscar.doctestsetup()
 
 Given a coefficient ring $C$ as in the previous section, let $C[x]=C[x_1, \ldots, x_n]$
 be the polynomial ring over $C$ in the set of variables $x=\{x_1, \ldots, x_n\}$. Monomials
-in $x=\{x_1, \ldots, x_n\}$ are written using multi--indices: If $\alpha=(\alpha_1, \ldots, \alpha_n)\in \N^n$,
+in $x=\{x_1, \ldots, x_n\}$ are written using multi--indices: If $\alpha=(\alpha_1, \ldots, \alpha_n)\in \mathbb N^n$,
```

Comment:
```suggestion
in $x=\{x_1, \ldots, x_n\}$ are written using multi--indices: If $\alpha=(\alpha_1, \ldots, \alpha_n)\in \mathbb{N}^n$,
```
I, personally, find it easier to reason about latex with all parentheses given. But no strong opinion